.github/workflows/cargo-publish.yml

```diff
@@ -13,12 +13,12 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     timeout-minutes: 30
     env:
       # Need up-to-date compilers for kernels
-      CC: gcc-12
-      CXX: g++-12
+      CC: clang-18
+      CXX: clang-18
```

Comment:
We aren't shipping binaries. Why do we care about speed of kernels in this job?

Reply:
This PR is not finished yet.

Reply:
Oh lol. Try using the "draft" feature of GitHub in the future. Otherwise it is hard to tell when you are finish.

Reply:
well, you can ping me on slack :)